x-pack/test/detection_engine_api_integration/security_and_spaces/tests/perform_bulk_action.ts

```diff
@@ -156,5 +145,51 @@ export default ({ getService }: FtrProviderContext): void => {
 
       expect(rulesResponse.total).to.eql(2);
     });
+
+    it('should set, add and delete tags in rules', async () => {
+      const ruleId = 'ruleId';
+      const tags = ['tag1', 'tag2'];
+      await createRule(supertest, log, getSimpleRule(ruleId));
+
+      const { body } = await postBulkAction()
+        .send({
+          query: '',
+          action: BulkAction.update,
+          updates: [
+            {
+              type: BulkActionUpdateType.set_tags,
+              value: [],
+            },
+            {
+              type: BulkActionUpdateType.add_tags,
+              value: tags,
+            },
```

Comment:
Can we test `set` and `add` updates separately? It is not completely clear what a test with two updates combined verifies.